src/renderer/services/Eth2HDWallet.ts

```diff
@@ -0,0 +1,28 @@
+import * as bip39 from 'bip39';
```

Comment:
import bip39 from 'bip39' ?

Reply:
Module '"/home/domagoj/NodeFactory/ChainGuardian/node_modules/bip39/types/index"' has no default export.

Reply:
oh, stupid libraries. Either enable syntetic default export in tsconfig or leave it like this